WebApi/Controllers/WorkplacesController.cs

```diff
@@ -15,9 +15,9 @@ public class WorkplacesController : BaseApiController
     {
 
         [HttpGet]
-        public async Task<IActionResult> GetAll()
+        public async Task<IActionResult> GetAll([FromQuery]GetWorkplaceListQueryRequest request)
         {
-            var result = await Mediator.Send(new GetWorkplaceListQueryRequest());
+            var result = await Mediator.Send(request);
```

Comment:
I have just edited this line but other changes are from feature/delete-vacation branch